department-of-veterans-affairs/caseflow
# Wmedders21/appeals 29427

Resolves [Intake Form Step 3: Confirmation - Tasks not related to an Appeal](https://jira.devops.va.gov/browse/APPEALS-29427)

# Description

- Adds 'Tasks not related to an Appeal' section on the confirm step (step 3) of the Correspondence Intake form
- When the user clicks the 'Edit section' link, they are redirected to step 2 of the form and automatically scrolled to the 'Tasks not related to an Appeal' section.

## Acceptance Criteria

## Testing Plan

Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below
1. On the intake form, click Continue to get to step 2.
2. Under "Tasks not related to an Appeal", click + Add tasks.
3. Select a task from the dropdown and enter sample text in the text box.
4. click Continue
5. On Step 3, expect to see a section "Tasks not related to an Appeal"
6. Expect a table with Columns titled "Tasks" and "Task Instructions or Context"
7. Expect the task and sample text from Step 2 to appear in the table
8. Click the "Edit section" link for "Tasks not related to an Appeal"
9. Expect the page is back on Step 2 and the view is scrolled to the "Tasks not related to an Appeal section"
  


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---
![Screenshot 2023-11-21 at 7 58 33 PM](https://github.com/department-of-veterans-affairs/caseflow/assets/93014155/ea76d184-651b-4a24-aebf-a611df932c4a) | ![Screenshot 2023-11-21 at 7 57 20 PM](https://github.com/department-of-veterans-affairs/caseflow/assets/93014155/eb2ffb69-32ce-4ef8-ab44-1c3bd4060179)

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
